src/sentry/deletions/defaults/project.py

```diff
@@ -61,6 +60,7 @@ def get_child_relations(self, instance):
                 for m in model_list
             ]
         )
+        relations.append(ModelRelation(Monitor, {"project_id": instance.id}))
```

Comment:
Why wasn't this able to be in the above model list?

Reply:
those are using `BulkModelDeletionTask` which runs raw sql and doesn't delete the related objects to the `Monitor`. think there's an issue somewhere i can dig up

Reply:
SENTRY-ZJW